augly/video/augmenters/ffmpeg/hflip.py

```diff
@@ -1,16 +1,13 @@
 #!/usr/bin/env python3
 # Copyright (c) Facebook, Inc. and its affiliates.
 
-from typing import Tuple, Dict
+from typing import List
 
 from augly.video.augmenters.ffmpeg.base_augmenter import BaseFFMPEGAugmenter
-from ffmpeg.nodes import FilterableStream
 
 
 class VideoAugmenterByHFlip(BaseFFMPEGAugmenter):
-    def add_augmenter(
-        self, in_stream: FilterableStream, **kwargs
-    ) -> Tuple[FilterableStream, Dict]:
+    def add_augmenter(self, video_path, output_path, **kwargs) -> List[str]:
```

Comment:
I'm not sure this is the best practice to override the BaseFFMPEGAugmenter class methods but change the output type (base is defined [here](https://github.com/facebookresearch/AugLy/blob/main/augly/video/augmenters/ffmpeg/base_augmenter.py#L54)). Are you planning to change **all** the video ffmpeg augmenter classes to behave like this? In that case it's probably better to change the base class itself. But if some will use the old pattern & some will use this new pattern because you're changing only some to use vidgear, then I would say you should define another base class. For example you could define it in the same file as BaseFFMPEGAugmenter but call it BaseVidgearFFMPEGAugmenter and define a method stub for add_augmenter with this function signature (returning a List[str]).

Reply:
Hmm, so the plan is to change all the video ffmpeg augmenter classes but I guess since that's not definite I think I will create a new base class for VidGear.